examples/04_pong/systems/paddle.rs

```diff
@@ -9,10 +9,12 @@ use amethyst::timing::Time;
 pub struct PaddleSystem;
 
 impl<'s> System<'s> for PaddleSystem {
-    type SystemData = (WriteStorage<'s, Paddle>,
-     WriteStorage<'s, LocalTransform>,
-     Fetch<'s, Time>,
-     Fetch<'s, InputHandler<String, String>>);
+    type SystemData = (
+        WriteStorage<'s, Paddle>,
```

Comment:
Somebody changed this before, now it's changed back. The question is whose formatting tool is the one we agreed on.

Reply:
I think @Binero might have forgotten to do it after the last changes of the Pong PR.